contracts/test/mocks/APMRegistryFactoryMock.sol

```diff
@@ -1,31 +1,49 @@
 pragma solidity 0.4.24;
```

Comment:
This mock has been refactored to be simpler, and more targeted about what's missing.